### PR TITLE
Professional icons

### PR DESCRIPTION
**Pull request by MikhailArkhipov**

### CODE REVIEW

File: src/Setup/Product.wxs [hunk 1]
```diff
@@ -161,7 +161,6 @@
                 <Directory Id="ItemTemplatesFolder" Name="ItemTemplates">
                   <Directory Id="ItemTemplatesInstallFolder" Name="R">
                     <Directory Id="ItemTemplatesInstallFolder_DatasetRD" Name="DatasetRD" />
-                    <Directory Id="ItemTemplatesInstallFolder_EmptyMD" Name="EmptyMD" />
```
Comment (AlexanderSher) [inline src/Setup/Product.wxs]: So they now part of RTVS package rather than template, correct?

Comment (MikhailArkhipov) [inline src/Setup/Product.wxs]: Icons - yes. I also removed plain markdown.

Comment (MikhailArkhipov) [inline src/Setup/Product.wxs]: Like `<Icon Package="{6D7C5336-C0CA-4857-A7E7-2E964EA836BF}" ID="503" />`

**Comment by AlexanderSher** [comment]
👍

**Pull request merged**